### PR TITLE
Fix/mibm k8s hc helm

### PR DESCRIPTION
**Pull request by bogdanguranda**
Healthcheck improvement.
Helm: pod anti-affinity, memory requests and limit.

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+              echo 'export DOCKER_TAG=latest' >> $BASH_ENV
+            else
+              echo 'export DOCKER_TAG=$( echo ${CIRCLE_BRANCH} | cut -f 2 -d / )' >> $BASH_ENV
+            fi
+      - run:
+          name: Build Docker image
+          command: |
+            source ${BASH_ENV}
+            docker build \
+            --build-arg SONATYPE_USER=${SONATYPE_USER} \
```
Comment (kghatton) [inline .circleci/config.yml]: These values are not consumed by the Dockerfile and should be removed.

File: helm/methode-image-binary-mapper/values.yaml [hunk 2]
```diff
@@ -8,8 +8,10 @@ replicaCount: 2
 image:
   repository: coco/methode-image-binary-mapper
   pullPolicy: IfNotPresent
-#resources:
-#  limits:
-#    memory: 256Mi
+resources:
+  requests:
+    memory: 360Mi
```
Comment (kghatton) [inline helm/methode-image-binary-mapper/values.yaml]: Looks a bit low when I look at Grafana, the average memory usage is between 420-440Mi.

File: pom.xml [hunk 3]
```diff
@@ -10,6 +10,7 @@
   <properties>
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
     <dw.version>0.7.1</dw.version>
+    <advanced-dropwizard-healthcheck.version>4.14.1</advanced-dropwizard-healthcheck.version>
```
Comment (kghatton) [inline pom.xml]: Needs to be updated to latest point release.

**Review by kghatton — CHANGES_REQUESTED** [review]

File: pom.xml (hunk 3 above)
Comment (bogdanguranda) [reply pom.xml]: Right, thx.

File: helm/methode-image-binary-mapper/values.yaml (hunk 2 above)
Comment (bogdanguranda) [reply helm/methode-image-binary-mapper/values.yaml]: That's right, I'll increase it.

File: .circleci/config.yml (hunk 1 above)
Comment (bogdanguranda) [reply .circleci/config.yml]: You are right. The majority of other CircleCI config.yml I saw are having this unused args for docker build.

**Review by kghatton — APPROVED** [review]

**Pull request merged**